packages/mobile/src/assets/images/iconCollectible.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="24" height="25" viewBox="0 0 24 25" fill="none" xmlns="http://www.w3.org/2000/svg">
+    <path d="M21.5398 2.78725C21.8288 3.02158 21.9967 3.37697 21.9967 3.75189V9.37569C21.9967 10.4106 20.8758 11.2503 19.4972 11.2503C18.1186 11.2503 16.9978 10.4106 16.9978 9.37569C16.9978 8.34075 18.1186 7.50109 19.4972 7.50109V5.275L15.748 6.02484V10.6254C15.748 11.6604 14.6272 12.5 13.2486 12.5C11.87 12.5 10.7491 11.6604 10.7491 10.6254C10.7491 9.59048 11.87 8.75082 13.2486 8.75082V5.00162C13.2486 4.40409 13.6703 3.89249 14.2523 3.77532L20.5009 2.52559C20.868 2.45139 21.2468 2.54902 21.5398 2.78335V2.78725ZM4.91832 14.3746L5.37916 13.6795C5.60958 13.3319 6.00012 13.1249 6.418 13.1249H8.82764C9.24552 13.1249 9.63606 13.3319 9.86648 13.6795L10.3312 14.3746H11.374C12.4089 14.3746 13.2486 15.2143 13.2486 16.2492V20.6233C13.2486 21.6582 12.4089 22.4979 11.374 22.4979H3.87557C2.84064 22.4979 2.00098 21.6582 2.00098 20.6233V16.2492C2.00098 15.2143 2.84064 14.3746 3.87557 14.3746H4.91832ZM9.49937 18.4362C9.49937 17.4013 8.6597 16.5616 7.62477 16.5616C6.58984 16.5616 5.75017 17.4013 5.75017 18.4362C5.75017 19.4712 6.58984 20.3108 7.62477 20.3108C8.6597 20.3108 9.49937 19.4712 9.49937 18.4362ZM20.6962 13.3709L19.1965 16.8741H21.3718C21.6335 16.8741 21.8639 17.0342 21.9576 17.2802C22.0514 17.5263 21.9811 17.7997 21.7858 17.9715L16.7869 22.3456C16.5682 22.5369 16.244 22.5525 16.0097 22.3807C15.7754 22.2089 15.6894 21.8964 15.8027 21.6309L17.2985 18.1238H15.1232C14.8615 18.1238 14.6311 17.9637 14.5373 17.7176C14.4436 17.4716 14.5139 17.1982 14.7092 17.0264L19.7081 12.6523C19.9268 12.461 20.251 12.4453 20.4853 12.6172C20.7196 12.789 20.8055 13.1014 20.6923 13.367L20.6962 13.3709ZM7.45684 11.0589C7.20299 11.3128 6.79292 11.3128 6.55859 11.0589L2.77815 7.15741C1.68464 6.02484 1.74322 4.15415 2.97342 3.09969C4.02789 2.18192 5.64473 2.35766 6.62108 3.35354L7.01162 3.76361L7.38263 3.35354C8.35899 2.35766 9.9563 2.18192 11.0498 3.09969C12.2605 4.15415 12.3191 6.02484 11.2256 7.15741L7.46074 11.0589H7.45684Z" fill="#000"/>
```

Comment:
Shouldn't these be red? Or are we supposed to make them black?

Reply:
hmm not sure why red. but the reason why we have `#000` is so that we can override the fill color. @dylanjeffers helped me figure out that we couldn't update the color because the fill was set to another non-black color and react native didn't like that